### PR TITLE
MOLIM-133 Fixed phone number provider to handle 0000 cases

### PR DESCRIPTION
**Pull request by tomjas1997**

### CODE REVIEW

File: tests/Unit/Provider/PhoneNumberProviderTest.php [hunk 1]
```diff
@@ -54,6 +54,14 @@
 //                'phoneNumber' => '+0164742671',
```
Comment (tomas862) [inline tests/Unit/Provider/PhoneNumberProviderTest.php:54]: please dont comment tests- why they are commented  ? If not needed remove them and replace with new ones

Comment (tomjas1997) [reply tests/Unit/Provider/PhoneNumberProviderTest.php:54]: They were commented to with commit "disabled unit tests till we find a way to make it work on github"
Adding those cases to be available with fix

Comment (tomjas1997) [reply tests/Unit/Provider/PhoneNumberProviderTest.php:54]: Whole test is commented due to not working on github

Comment (tomas862) [reply tests/Unit/Provider/PhoneNumberProviderTest.php:54]: arrg, its not working due to it does not find PRestaShop global entity classes, or any clases from PRestaShop.

You can try adding this in unit test

```php

if (!class_exists(\Validate)) { 
  class Validate { 
    // copy methods from Presta that you use in test
   
  }

 }

```

Comment (tomjas1997) [reply tests/Unit/Provider/PhoneNumberProviderTest.php:54]: i do not have tests configured for this mollie :(

**Review by tomas862 — APPROVED** [review]
I dont like that tests are commented, need to fix this ASAP in next releases

**Pull request merged**